Documentation/installation/requirements-aks.rst

```diff
@@ -7,8 +7,14 @@ BYOCNI (``none``)             Encapsulation   Cluster Pool        Kubernetes CRD
 Legacy Azure IPAM (``azure``) Direct Routing  Azure IPAM          Kubernetes CRD
 ============================= =============== =================== ==============
 
-Using `Bring your own CNI <https://docs.microsoft.com/en-us/azure/aks/use-byo-cni?tabs=azure-cli>`_
-is the preferred way to run Cilium on `Azure Kubernetes Service (AKS) <https://docs.microsoft.com/en-us/azure/aks/>`_,
-however integration with the Azure stack via the :ref:`Azure IPAM<ipam_azure>`
-is not available and will only work with clusters not using BYOCNI. While still
-maintained for now, this mode is considered legacy.
+The preferred way to run Cilium on `Azure Kubernetes Service (AKS) <https://docs.microsoft.com/en-us/azure/aks/>`_ is 
```

Comment:
I'm also wondering if we should just drop the "preferred" step here and walk this back to just a statement that "AKS provides two methods to integrate Cilium CNI" and point to the two, then explain about the benefits in a minimal comparative way.